src/mol-plugin-state/transforms/representation.ts

```diff
@@ -529,6 +533,121 @@ const TransparencyStructureRepresentation3DFromBundle = PluginStateTransform.Bui
     }
 });
 
+type SubstanceStructureRepresentation3DFromScript = typeof SubstanceStructureRepresentation3DFromScript
```

Comment:
Add State to the name ``SubstanceStructureRepresentationState3DFromScript`` ?

Reply:
Hmm, not sure, if we do that we should change all the other transforms that create a Representation3DState as well.

Reply:
Fair point.